Microsoft/WebTemplateStudio
# angular fix ng build prod errors

**How to test:**
1. generate a angular project with 4 types of page selected 
2. do npm install
3. do ng build --prod, there should be no errors 

To address one suggestion from Angular Expert #681 

Errors to be addressed by this PR:
![ngbuild](https://user-images.githubusercontent.com/12874317/59543870-85bbfc80-8ec2-11e9-8401-3e9238966be1.PNG)
